crates/nu-protocol/src/shell_error.rs

```diff
@@ -1006,6 +1006,11 @@ pub enum ShellError {
     #[diagnostic(code(nu::shell::non_unicode_input))]
     NonUnicodeInput,
 
+    /// Sleep is breaked.
+    #[error("Sleep is breaked.")]
```

Comment:
How about something more like this
```suggestion
    #[error("CTRL-C pressed during sleep command.")]
```

Reply:
Cool!  I should search variants of `ShellError` carefully before adding a new one

Reply:
Thanks @WindSoilder !